components/dada-lex/src/lex.rs

```diff
@@ -203,6 +215,19 @@ where
                 break;
             }
 
+            if ch == '\\' {
```

Comment:
I think this should be done in a later phase, after lexing-- in the parser or during validation, for example. In particular, I think that explicitly written `\n` and `\t` etc should never be stripped by the whitespace normalization code.

Reply:
Got it, I'll fix it later.

Reply:
I modified it, feel free to request changes.